app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsViewModel.kt

```diff
@@ -190,6 +190,7 @@ class TransactionDetailsViewModel
                     credentialsRepository.signWithOwner(owners[0], executionInfo.safeTxHash.hexToByteArray())
                 )
             }.onSuccess {
+                txDetails = it
```

Comment:
This doesn't seem to be enough. The button is still there until I pull to refresh.